### PR TITLE
[TR] Man page translations

### PR DESCRIPTION
**Pull request by bitigchi**
This commit includes man page translations for Turkish Vim.

### CODE REVIEW

**Comment by bitigchi** [comment]
Folks, I was unable to get `man vim` working in Turkish. Am I missing something?

File: runtime/doc/vim.1 [hunk 1]
```diff
@@ -326,7 +326,7 @@ If you do want to overwrite a file, add an exclamation mark to the Ex command,
 as in ":w!".
 The \-R option also implies the \-n option (see above).
 The 'readonly' option can be reset with ":set noro".
-See ":help 'readonly'".
+See ":help readonly".
```
Comment (k-takata) [inline runtime/doc/vim.1]: This change should be reverted.

**Comment by k-takata** [comment]
> Folks, I was unable to get `man vim` working in Turkish. Am I missing something?

You may need to update `src/Makefile`. Search `MAN1DIR` and `DEST_MAN`.

**Comment by bitigchi** [comment]
Commit 3fc6b2f causes a build failure.

**Comment by brammool** [comment]
> Commit [3fc6b2f](https://github.com/vim/vim/commit/3fc6b2f81ca5df12543c55409411f604e1351692) causes a build failure.
This is not helpful.  What commit is this referring to?  What failure?

**Comment by bitigchi** [comment]
> Commit [3fc6b2f](https://github.com/vim/vim/commit/3fc6b2f81ca5df12543c55409411f604e1351692) causes a build failure.
> This is not helpful. What commit is this referring to? What failure?

Oh, it's my last commit, modifications for the Makefile. I'm trying to figure out where it fails. Error output below:
> Undefined symbols for architecture x86_64:
  "_add_time", referenced from:
      _buflist_list in buffer.o
      _u_undo_end in undo.o
      _ex_undolist in undo.o
     (maybe you meant: _did_add_timer)
  "_check_due_timer", referenced from:
      _do_sleep in ex_docmd.o
      _ui_wait_for_chars_or_timer in ui.o
  "_create_timer", referenced from:
      _popup_create in popupwin.o
      _popup_add_timeout in popupwin.o
  "_f_localtime", referenced from:
      _global_functions in evalfunc.o
  "_f_reltime", referenced from:
      _global_functions in evalfunc.o
  "_f_reltimefloat", referenced from:
      _global_functions in evalfunc.o
  "_f_reltimestr", referenced from:
      _global_functions in evalfunc.o
  "_f_strftime", referenced from:
      _global_functions in evalfunc.o
  "_f_strptime", referenced from:
      _global_functions in evalfunc.o
  "_f_timer_info", referenced from:
      _global_functions in evalfunc.o
  "_f_timer_pause", referenced from:
      _global_functions in evalfunc.o
  "_f_timer_start", referenced from:
      _global_functions in evalfunc.o
  "_f_timer_stop", referenced from:
      _global_functions in evalfunc.o
  "_f_timer_stopall", referenced from:
      _global_functions in evalfunc.o
  "_get8ctime", referenced from:
      _spell_load_file in spellfile.o
      _suggest_load_files in spellfile.o
      _u_read_undo in undo.o
  "_get_ctime", referenced from:
      _mch_print_begin in hardcopy.o
      _findswapname in memline.o
      _swapfile_info in memline.o
  "_proftime_time_left", referenced from:
      _term_check_timers in terminal.o
  "_put_time", referenced from:
      _mkspell in spellfile.o
  "_set_ref_in_timer", referenced from:
      _garbage_collect in eval.o
  "_stop_timer", referenced from:
      _win_free_popup in window.o
  "_time_msg", referenced from:
      _do_source in scriptfile.o
      _main in main.o
      _common_init in main.o
      _vim_main2 in main.o
      _main_loop in main.o
  "_time_pop", referenced from:
      _do_source in scriptfile.o
  "_time_push", referenced from:
      _do_source in scriptfile.o
  "_time_to_bytes", referenced from:
      _u_write_undo in undo.o
  "_vim_time", referenced from:
      _enter_buffer in buffer.o
      _in_history in cmdhist.o
      _add_to_history in cmdhist.o
      _f_rand in evalfunc.o
      _f_srand in evalfunc.o
      _do_ecmd in ex_cmds.o
      _setmark_pos in mark.o
      ...
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [vim] Error 1

**Comment by bitigchi** [comment]
@brammool, I am still not sure what I failed to do. I've also checked previous commits regarding new man pages, and it checks out; but still Vim fails to build with new additions. Any ideas?

File: src/Makefile [hunk 2]
```diff
@@ -2599,6 +2609,13 @@ install-languages: languages $(DEST_LANG) $(DEST_KMAP)
 		$(DEST_MAN_RU) $(INSTALLMLARGS)
 	-$(SHELL) ./installml.sh install "$(GUI_MAN_TARGETS)" \
 		$(DEST_MAN_RU_U) $(INSTALLMLARGS)
+	-$(SHELL) ./installml.sh install "$(GUI_MAN_TARGETS)" \
+		$(DEST_MAN_TR) $(INSTALLMLARGS)
+	-$(SHELL) ./installml.sh install "$(GUI_MAN_TARGETS)" \
+		$(DEST_MAN_TR_I) $(INSTALLMLARGS)
+	-$(SHELL) ./installml.sh install "$(GUI_MAN_TARGETS)" \
+		$(DEST_MAN_TR_U) $(INSTALLMLARGS)
+
```
Comment (k-takata) [inline src/Makefile]: Don't add a blank line here.

**Comment by k-takata** [comment]
> Oh, it's my last commit, modifications for the Makefile. I'm trying to figure out where it fails. Error output below:

Time and timer functions are moved into time.c by a recent patch.
You should merge the latest master branch or rebase on top of it.

**Comment by bitigchi** [comment]
> > Oh, it's my last commit, modifications for the Makefile. I'm trying to figure out where it fails. Error output below:
> 
> Time and timer functions are moved into time.c by a recent patch.
> You should merge the latest master branch or rebase on top of it.

~~I am already basing on the latest patch.~~

Yes, I realised that the Makefile in my branch was outdated. Sorry about that, my bad.

File: src/Makefile (hunk 2 above)
Comment (bitigchi) [reply src/Makefile]: Fixed.

**Comment by bitigchi** [comment]
It launches fine, but on macOS Turkish characters are not properly displayed, Linux does not have this problem. I suspect this is a macOS problem, so it's ready to include now. I'll see if it can be fixed on macOS, any fixes are of course welcome.

**Comment by k-takata** [comment]
I wonder if we need the files in a legacy encoding (latin5) these days.
FYI, Japanese man pages have only UTF-8 files.

**Comment by bitigchi** [comment]
> I wonder if we need the files in a legacy encoding (latin5) these days.
> FYI, Japanese man pages have only UTF-8 files.

Most intriguing, I deleted the conversions, latin5 versions of the files, and left only `DEST_MAN_TR_U` entries in Makefile, but characters still do not display properly, they are displaying with their latin1 correspondents. Most likely a macOS issue.

**Pull request closed without merging**